docs/changelog.rst

```diff
@@ -17,6 +17,7 @@ Please refer to :doc:`release notes<release_notes>`.
 - Clarified ``skip_diff`` documentation (#1655)
 - Standardised interface of :meth:`~import_export.widgets.Widget.render` (#1657)
 - Improved documentation relating to validation on import (#1665)
+- Refactored test_admin_integration: split into smaller test modules (# 1662)
```

Comment:
please remove the whitespace so that the line renders correctly i.e. change (# 1662) to (#1662)